src/main/java/com/jetbrains/snakecharm/string_language/lang/parser/_SmkSLLexer.java

```diff
@@ -0,0 +1,552 @@
+/* The following code was generated by JFlex 1.7.0 tweaked for IntelliJ platform */
```

Comment:
Please 
1. remove generated class from git repo
2. add it to git ignore file
3. updated `build.gradle` to include code generation step before compilation. I think `https://github.com/thomaslee/gradle-jflex-plugin` will help

Reply:
SmkSL lexer is built with a tweaked version of JFlex to make it compatible with IDEA. So it is easier to leave the generated class as is, furthermore it seems to be a normal practice to keep generated files in repo for other projects.